app/src/main/java/com/dimelo/sampleapp/RcSourceAdaptater.java

```diff
@@ -0,0 +1,87 @@
+package com.dimelo.sampleapp;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.TextView;
+import java.util.List;
+
+public class RcSourceAdaptater extends RecyclerView.Adapter<RcSourceAdaptater.ViewHolder> {
```

Comment:
You mean Adapter, right? 

```suggestion
public class RcSourceAdapter extends RecyclerView.Adapter<RcSourceAdaptater.ViewHolder> {
```